lib/molecules/admin-bar/template.html

```diff
@@ -0,0 +1,25 @@
+<div class="toolbar" role="navigation" aria-label="Toolbar" tabindex="0" ng-if="vm.adminBarEnabled">
```

Comment:
Yes, it's done also in the update-patternlab branch of ng-web-app: https://github.com/moxie-lean/ng-web-app/blob/update-patternlab/app/patternlab/molecules/_ln-m-admin-bar.scss